tests/service/data/CSharp_Analysis/CSharp_Analysis.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard20</TargetFramework>
```

Comment:
Good catch

Reply:
`NETSTANDARD20` is the compiler constant introduced by MSBuild; `netstandard2.0` is still the correct TFM.